cmd/otelcontribcol/go.mod

```
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.21.0
 
-toolchain go1.21.12
+toolchain go1.22.3
```

You may need to run make commands with `GOTOOLCHAIN=go1.21.12 make ...`

```suggestion
toolchain go1.21.12
```

---

@djaglowski Tried this. CI still seems to have issues.

---

Nevermind...after the latest rebase with main seems fine all of a sudden.